### PR TITLE
my first component; FIX #4

### LINKED ISSUE #4: Create simple react component

### PR DESCRIPTION
**Pull request by alex-koshara**
#4

### CODE REVIEW

**Comment by sergeyamator** [comment]
Travis has failed

**Comment by alex-koshara** [comment]
I dont know this problem

File: src/Components/TestComponent/index.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './style.scss';
+
+const testComponent = props => (
```
Comment (sergeyamator) [inline src/Components/TestComponent/index.js]: Component should start with uppercase letter (TestComponent) only

Comment (sergeyamator) [reply src/Components/TestComponent/index.js]: Otherwise it will not work

**Pull request closed without merging**